client/Overview/DescriptionBanner.jsx

```diff
@@ -17,16 +17,19 @@ const DescriptionBanner = (props) => {
       </div>
       <div id="descriptionAreaRight">
         <ul>
-          {/* {selectProductInfo.features.map(item => {
+          {selectProductInfo.features.map((item) => (
             <li key={item.feature} className="checkIt">
-              <h3>✓</h3>
-              <span>{item.feature}</span>
+              <span>
+                ✓
+                {item.feature}
```

Comment:
If `item.feature` is a string, you can condense this with template literals.

```
{`✓ ${item.feature}`}
```

It's not super necessary in this case, but in the future, if you have more than two lines of this, it's a neat trick to know.